tasks/Resources/SERH_OLM/install.resource

```diff
@@ -0,0 +1,145 @@
+*** Settings ***
+Documentation    Collections of keywords to install/uninstall Managed Starburst
+Library          OpenShiftLibrary
+Resource         ../../../tests/Resources/Common.robot
+
+
+*** Variables ***
+${STARBURST_CR_DEFAULT_NAMESPACE}=    redhat-starburst-operator
+
+
+*** Keywords ***
+Check Managed Staburst Is Not Installed
+    [Documentation]    Checks Managed Staburst is Not installed on the cluster.
+    ...                Fails if it is
+    ${installed}=    Is Managed Starburst Installed
+    IF    ${installed} == ${TRUE}
+        Fail    msg=Managed Starburst is still installed or not fully Uninstalled
+    END
+
+Is Managed Starburst Installed
+    [Documentation]    Checks Managed Staburst is installed on the cluster.
+    ...                Returns TRUE if the resources created by the install process
+    ...                are found in the cluster
+    ${result}=  Run Keyword And Return Status
+    ...  Run Keywords
+    ...  Oc Get  kind=Project  field_selector=metadata.name=${STARBURST_CR_DEFAULT_NAMESPACE}  AND
+    ...  Oc Get  kind=CatalogSource  namespace=${STARBURST_CR_DEFAULT_NAMESPACE}
+    ...    field_selector=metadata.name=addon-managed-starburst-catalog
```

Comment:
Isn't this check enough? If this CS exists then the NS/Project must exist as well

Reply:
hm probably yes? I mean, I expect during installation that the addon first creates the NS and then the CS. But in general, a CS can exist without the NS if i'm not mistaken. You could try deleting a NS and checking that the CS remains